content/blog/2024/09/kaoto-release-2.2.0/index.md

```diff
@@ -0,0 +1,104 @@
+
+<div style="text-align: center;">
+  <img src="componentIcons.png" alt="Component icons" height="400"/>
+</div>
+
+- Pagination for the catalog: The catalog now supports pagination, making it easier and faster to navigate through the components and find the one you need.
+
+<div style="text-align: center;">
+  <img src="catalogPagination.png" alt="Catalog pagination" height="400"/>
+</div>
+
+- Warning when replacing steps: When replacing a step in the canvas, you will now get a warning if the step you are replacing has child steps, preventing you from losing your work.
+
+<div style="text-align: center;">
+  <img src="warningWhenReplacingSteps.png" alt="Catalog pagination" height="400"/>
+</div>
+
+- Choose the label to display: You can now choose whether to use the step id or the step description for the step labels, offering more flexibility in how you work with the canvas.
+
+<div style="text-align: center;">
+  <img src="chooseNodeLabel.png" alt="Choose node label"/>
+</div>
+
+- Remember side bar width: The side panel width is now remembered between sessions, making it easier to work with the side panel.
+
+- Expose models through the NPM package: You can now use the Kaoto models in your own projects by installing the Kaoto NPM package, making easier to integrate Kaoto in your own projects.
+
+## Let's build it together
+
+Let us know what you think by joining us in the [GitHub discussions](https://github.com/orgs/KaotoIO/discussions). Do you have an idea how to improve Kaoto? Would you love to see a useful feature implemented or simply ask a question? Please do so [here](https://github.com/KaotoIO/kaoto/issues/new/choose).
+
+You can also join the [Kaoto channel](https://camel.zulipchat.com/#narrow/stream/441302-kaoto) on the Camel Zulip Chat.
+
+You can also check out our [examples repository](https://github.com/KaotoIO/kaoto-examples) to see some more advanced use cases which were set up using Kaoto.
+
+<div style="text-align: center">
+  <ins>All contributions are welcome!</ins>
+</div>
+
+## What comes next?
+
+Kaoto 2.2 is a step forward in our commitment to prove a powerful and flexible visual integration platform. We’ll continue to focus on improving the usability, adding new features, and making Kaoto the best tool for your integration needs.
+
+Kaoto continues to be available as [a VS Code extension](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-kaoto), ensuring seamless integration into your development environment. We also host [a showcase environment](https://kaotoio.github.io/kaoto/) accessible in your browser for easy exploring.
+
+Feel free to explore the new features and improvements, and let us know what you think!
+
+---
+<br/><br/>
+<div style="text-align: center">
+We invite you to explore these new features on your summer integrations.
```

Comment:
summer integrations? is it a left-over of a copy-paste?

Reply:
Yes, my bad, I rephrased it